### PR TITLE
Document AoU-specific tieout [VS-233]

### PR DESCRIPTION
**Pull request by rsasch**

### CODE REVIEW

File: scripts/variantstore/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -9,37 +9,38 @@
...
-        - if the callset has more than 20K genomes, you will want to set the attribute skip_vet_new_insert to true which will run all of the queries except the last one. It will print out the last query that needs to run with flex slots. Allocate the flex slots, run the query, and then unallocate them. 
+        - If the ```sample_info``` table contains control samples, create views in your dataset called ```sample_info_without_controls``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id > 50``` and ```sample_info_controls_only``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id <= 50```. A run for ```sample_info_without_controls``` will create the tables for the joint callset deliverable.  The run with ```sample_info_controls_only```  (with a distinct `destination_cohort_table_prefix` from the non-control run) will be used for the ["Sensitivity and Precision" deliverable](tieout/AoU_PRECISION_SENSITIVIT.md).
```
Comment (RoriCremer) [inline scripts/variantstore/AOU_DELIVERABLES.md:14]: (tieout/AoU_PRECISION_SENSITIVITY.md).  ?
are you missing a "y" ?

File: scripts/variantstore/AOU_DELIVERABLES.md [hunk 2]
```diff
@@ -9,37 +9,38 @@
...
+        - If the callset has more than 20K genomes, you will want to set the input "skip_vet_new_insert" to `true` which will run all of the queries except the last one. It will print out the last query that needs to run with flex slots; allocate the flex slots, run the query, and then deallocate them.
```
Comment (RoriCremer) [inline scripts/variantstore/AOU_DELIVERABLES.md:15]: are we going to give more in depth instructions of allocating and deallocating flex slots somewhere else? or is it assumed that people will know how to do this?

File: scripts/variantstore/AOU_DELIVERABLES.md [hunk 3]
```diff
@@ -9,37 +9,38 @@
...
     6. GvsExtractCallset
-        - include a valid (and secure) “output\_gcs_dir” parameter that the service account (see above) has access to, this is where the VCF and interval list files  will go
-        - make note of the “cohort\_extract\_table_prefix” input that you enter
-    7. Run [callset QC](callset_QC/README.md), for which you need
+        - include a valid (and secure) "output_gcs_dir" parameter that the service account (see above) has access to, this is where the VCF and interval list files  will go
+        - use the last part of "fq_cohort_extract_table_prefix" from `GvsPrepareCallset` for the input "extract_table_prefix"
```
Comment (RoriCremer) [inline scripts/variantstore/AOU_DELIVERABLES.md:18]: what's the last part?

Comment (RoriCremer) [reply scripts/variantstore/AOU_DELIVERABLES.md:18]: maybe an example would be helpful?

**Review by RoriCremer — APPROVED** [review]
just some small nits

File: scripts/variantstore/AOU_DELIVERABLES.md (hunk 3 above)
Comment (rsasch) [reply scripts/variantstore/AOU_DELIVERABLES.md]: changed the wording, hopefully clearer now

File: scripts/variantstore/AOU_DELIVERABLES.md (hunk 2 above)
Comment (rsasch) [reply scripts/variantstore/AOU_DELIVERABLES.md:15]: Since the prepare step is going away, it's probably not worth it to go into flex slots right now.

**Pull request merged**